examples/generics/src/index.ts

```diff
@@ -0,0 +1,218 @@
+import { Alias, nat64, $query, Record, Result, Variant } from 'azle';
+
+// TODO don't forget tuples
```

Comment:
Don't forget tuples

Reply:
Oh wait, is this addressed with https://github.com/demergent-labs/azle/pull/984?